packages/devtools_app/lib/src/framework/html_framework.dart

```diff
@@ -687,17 +690,17 @@ class HtmlToastAnimator {
 
   static const Duration extendedHideDelay = Duration(seconds: 10);
 
+  static const Duration infiniteHideDelay = Duration(days: 1);
```

Comment:
nit: why not double.infinite for the number of days?
https://api.dartlang.org/stable/2.5.0/dart-core/double/infinity-constant.html